src/main/java/carRacing/CarMain.java

```diff
@@ -0,0 +1,23 @@
+package carRacing;
+
+import java.util.List;
+
+public class CarMain {
+
+	public static void main(String[] args) {
+		InputView input = new InputView();
+		int carCount = input.inputCarCount();
+		int moveCount = input.inputMoveCount();
+
+		System.out.println("실행 결과");
+
+		ResultView result = new ResultView();
+		CarRacing carRacing = new CarRacing();
+		List<Car> cars = carRacing.makeCars(carCount);
```

Comment:
```suggestion
		CarRacing carRacing = new CarRacing(carCount)
```
`List<Car>` 을 래핑하고 컬렉션과 관련된 로직을 CarRacing이 처리하고 있는데
이러한 객체를 일급컬렉션이라고 합니다.

하지만 위 코드에서는 일급컬렉션 활용이 되지 않는 것 같습니다.

위 처럼 생성자를 호출하여 생성과 동시에 내부 자동차 리스트를 초기화하면
일급컬렉션의 장점이 더 드러나지 않을까요?
```java
public CarRacing (int carCount) {
    // cars 리스트 초기화
} 
```